psi_cardinality/c/BUILD

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright 2020 the authors listed in CONTRIBUTORS.md
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "psi_cardinality_client_c",
+    srcs = ["psi_cardinality_client_c.cpp", "internal_utils.h", "internal_utils.cpp"],
```

Comment:
Since internal_utils is used in both the server and the client, I think we should move it to its own cc_library. You can set `visibility = ["//visibility:private"]` to ensure it's not accessible from the outside.